region/norfair/west/Ice Beam Gate Room.json

```diff
@@ -551,6 +551,149 @@
       "link": [6, 5],
       "name": "Base",
       "requires": []
+    },
+    {
+      "link": [7, 2],
+      "name": "Base",
+      "requires": [
+        {"obstaclesCleared": ["C"]},
+        {"or": [
+          "canWalljump",
+          "HiJump",
+          "h_canFly",
+          "canSpringBallJumpMidAir"
+        ]}
+      ]
+    },
+    {
+      "link": [7, 2],
+      "name": "Shinespark",
+      "requires": [
+        {"obstaclesCleared": ["C", "D"]},
+        "SpeedBooster",
+        {"or": [
+          {"shinespark": {"frames": 19, "excessFrames": 4}},
+          {"and": [
+            "canShinechargeMovementComplex",
+            "canMidairShinespark",
+            {"shinespark": {"frames": 6, "excessFrames": 4}}
+          ]}
+        ]}
+      ]
+    },
+    {
+      "link": [7, 2],
+      "name": "Ice Beam Gate Room Frozen Sova Platform",
+      "notable": true,
+      "requires": [
+        "h_canUsePowerBombs",
+        "canTrickyUseFrozenEnemies"
+      ],
+      "clearsObstacles": ["A", "B", "C"],
+      "note": [
+        "Place a Power Bomb to break the Power Bomb blocks and the bomb blocks, without killing the global Sova.",
+        "Wait 30-50 seconds for the Sova to get into position to be used as a platform."
+      ],
+      "devNote": "This is only notable with wall jumps disabled."
+    },
+    {
+      "link": [7, 3],
+      "name": "Enemies Already Killed",
+      "requires": [
+        {"obstaclesCleared": ["D"]}
+      ]
+    },
+    {
+      "link": [7, 3],
+      "name": "Enemy Kill",
+      "requires": [
+        {"or": [
+          "Spazer",
+          "Plasma",
+          "ScrewAttack",
+          "Wave",
+          {"and": [
+            "canDodgeWhileShooting",
+            "h_canUsePowerBombs"
+          ]},
+          {"enemyKill": {
+            "enemies": [["Sm. Dessgeega"], ["Sm. Dessgeega"]],
+            "explicitWeapons": [
+              "Missile",
+              "Super"
+            ]
+          }},
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyDamage": {
+              "enemy": "Sm. Dessgeega",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]},
+          {"enemyDamage": {
+            "enemy": "Sm. Dessgeega",
+            "type": "contact",
+            "hits": 2
+          }}
+        ]}
+      ],
+      "clearsObstacles": ["D"]
+    },
+    {
+      "link": [7, 3],
```

Comment:
There is a mockball strat to avoid all enemies.
It is solidly `Hard`, but I don't see too many people do it in randomizers.